content/en/docs/components/pipelines/v2/compile-a-pipeline.md

```diff
@@ -1,104 +1,75 @@
 
-**default_pipeline_root**  
-[`default_pipeline_root`][default-pipeline-root-schema] is the remote storage root path where the pipeline outputs will be written, such as a Google Cloud Storage URI (`gcs://my/path`).
+The compiled IR YAML file contains the following sections:
 
-[Example][default-pipeline-root-example]  
+| Section | Description | Example |
+|-------|-------------|---------|
+| [`components`][components-schema] | This section is a map of the names of all components used in the pipeline to [`ComponentSpec`][component-spec]. `ComponentSpec` defines the interface, including inputs and outputs, of a component.<br/>For primitive components, `ComponentSpec` contains a reference to the executor containing the component implementation.<br/>For pipelines used as components, `ComponentSpec` contains a [DagSpec][dag-spec] instance, which includes references to the underlying primitive components. | [View on Github][components-example]
+| [`deployment_spec`][deployment-spec-schema] | This section contains a map of executor name to [`ExecutorSpec`][executor-spec]. `ExecutorSpec` contains the implementation for a primitive component. | [View on Github][deployment-spec-example]
+| [`root`][root-schema] | This section defines the steps of the outermost pipeline definition, also called the pipeline root definition. The root definition is an instance of [`ComponentSpec`][component-spec], which is also the pipeline executed on Kubernetes when you submit the IR YAML definition to Kubernetes. | [View on Github][root-example]
+| [`pipeline_info`][pipeline-info-schema] <a id="kfp_iryaml_pipelineinfo"></a> | This section contains pipeline metadata, including the `pipelineInfo.name` field. This field contains the name of your pipeline template. When you upload your pipeline, a pipeline context name is created based on this template name. The pipeline context lets the backend and the dashboard associate artifacts and executions from pipeline runs using the pipeline template. You can use a pipeline context to determine the best model by comparing metrics and artifacts from multiple pipeline runs based on the same training pipeline. | [View on Github][pipeline-info-example]  
+| [`sdk_version`][sdk-version-schema] | This section records the version of the KFP SDK used to compiled the pipeline. | [View on Github][sdk-version-example]  
+| [`schema_version`][schema-version-schema] | This section records the version of the `PipelineSpec` schema used for the IR YAML. | [View on Github][schema-version-example]
+| [`default_pipeline_root`][default-pipeline-root-schema] | This section records the remote storage root path, such as a Google Cloud Storage URI, where the pipeline output is written. | [View on Github][default-pipeline-root-example]  
```

Comment:
nit/not your work:

```suggestion
| [`default_pipeline_root`][default-pipeline-root-schema] | This section records the remote storage root path, such as a MinIO URI or Google Cloud Storage URI, where the pipeline output is written. | [View on Github][default-pipeline-root-example]  
```

I originally only included GCS URI in this text. I think we should include Minio since Minio is the storage service that comes out-of-the-box with KFP.

Reply:
Following up on this.